### ISSUE TITLE
Deprecation warning on setAttachment in version 7.8.4 but new method only exists in 8.0.0

### ISSUE THREAD

**Issue by rounak** [Instabug/Instabug-iOS#234]
### Steps to Reproduce the Problem
Install v7.8.4, paste in the code below

### Expected Behavior
Code compiles without warnings, or that alternative method suggested exists in same version

### Actual Behavior
Emits deprecation warning, which if you have enabled warnings as errors results in error:
> Starting from v8.0, use `setAttachmentOptions:` instead

### Instabug integration code

```objc
[Instabug setAttachmentTypesEnabledScreenShot:YES
                                  extraScreenShot:YES
                                     galleryImage:YES
                                        voiceNote:YES
                                  screenRecording:YES];
```

### SDK Version
7.8.4

### iOS Version
10.3.1

### Device Type
Simulator

**Comment by Kmohamed** [comment]
@rounak We are very sorry for this mistake we will fix it in the next release.
But you could use this method instead

  `[Instabug setEnabledAttachmentTypes:IBGAttachmentTypeScreenShot | IBGAttachmentTypeGalleryImage | IBGAttachmentTypeExtraScreenShot | IBGAttachmentTypeScreenRecording];`

**Comment by rounak** [comment]
@Kmohamed thanks for the quick reply, will try that out

**Comment by Kmohamed** [comment]
Fix is released on v 7.9.1 Thanks @rounak 🙏
